### PR TITLE
fix: show keep_crv fee [WEB-413] and [WEB-408]

### PR DESCRIPTION
**Pull request by patcito**
Edit: fixed Not sure how to truncate keep_crv amount, let me know and I will update the PR. //cc @dudesahn 

I've also included the fix for WEB-408 as they both rely on the same fix.

### CODE REVIEW

**Comment by dudesahn** [comment]
`keepCRV` for triCrypto is 0 (which I caught thanks to this PR), but it would be great for any == 0 that we still display the value as equal to 0%

Screen Shot 2021-06-22 at 11 24 43 AM

**Comment by dudesahn** [comment]
`cvx_apr` values should be displayed as a percentage. So, a `cvx_apr` value of 0.07625203502738 would be 7.63%.

Screen Shot 2021-06-22 at 11 29 41 AM

**Comment by dudesahn** [comment]
One last fix: only show `keep_crv` for Curve vaults. So, should be able to do a check that apy type is `crv`

**Comment by patcito** [comment]
ok, let me know if it's ok now.

**Comment by dudesahn** [comment]
LGTM @patcito, I'll just defer to @x48-crypto to double-check the code itself

**Review by dudesahn — APPROVED** [review]

File: app/components/Vault/index.js [hunk 1]
```diff
@@ -605,10 +610,15 @@ const Vault = (props) => {
...
     if (v2Vault) {
       const { managementFee, performanceFee } = vault.fees.general;
-      const keepCrv =
-        vault.fees && vault.fees.special.keepCrv > 0
-          ? vault.fees.special.keepCrv
+      let keepCrv =
+        vault.apy &&
+        vault.apy.fees &&
+        vault.apy.fees.keep_crv &&
+        vault.apy.fees.keep_crv >= 0
```
Comment (x48-crypto) [inline app/components/Vault/index.js:617]: could probably replace with `_.get(vault, 'apy.fees.keep_crv')` but this is also find and i understand not wanting to rely on lodash

**Review by x48-crypto — APPROVED** [review]

**Pull request merged**